src/mdm-profile.resource.ts

```diff
@@ -120,6 +120,27 @@ export class MdmProfileResource extends MdmResource {
     return this.simpleGet(url, query, options);
   }
 
+  /**
+   * `HTTP GET` - Gets a profile definition for a namespace/name. This is similar to a full profile but
+   * without any assigned values or mapped to a catalogue item.
```

Comment:
I'm a little confused by this sentence. Do you mean, "... or _being_ mapped to a catalogue item"?